src/applications/disability-benefits/all-claims/tests/automated-tests.puppeteer.spec.js

```diff
@@ -32,8 +33,25 @@ const testConfig = {
       await page.waitFor('.usa-button-primary:not(.schemaform-start-button)');
       await page.click('.usa-button-primary');
     },
-    // TODO: Add a hook for the rated disabilities page
-    // TODO: Add a hook for the bank info page
+    '/disability/file-disability-claim-form-21-526ez/disabilities/rated-disabilities': async page => {
+      await page.click('input[name="root_ratedDisabilities_0"]');
+      await page.click('.form-progress-buttons .usa-button-primary');
+    },
+    '/disability/file-disability-claim-form-21-526ez/payment-information': async (
+      page,
+      data,
+      config,
+      log,
+    ) => {
+      // click to open edit mode for existing banking info if applicable
+      if (await page.$('.usa-button-primary.edit-button')) {
+        await page.click('.usa-button-primary.edit-button');
```

Comment:
I think we probably need to be a little more nuanced with the way we handle this page.

| Has pre-filled bank info | Has new bank info | Action |
| --- | --- | --- |
| No | No | Hitting continue should fail because of validation |
| No | Yes | Fill out the form (don't hit edit) |
| Yes | No | Continue without filling out the form |
| Yes | Yes | Hit edit and fill out the form |

Reply:
@cvalarida We certainly can add logic here to intelligently fill out or skip the bank info section depending on what is in the data files and whether payment info is pre-filled or not, e.g. case 2,3,4

It seems case 1 should fail outright in our test here, which primarily is for testing submission instead of omission of data. Anything you are thinking of for that failure case? From my perspective, either the test data files or the payment info endpoint should have bank account data available so we wouldn't expect to see the case where none of it is available.

Reply:
Right, so our test data should represent valid submissions (for now), so if the test data doesn't have it, the test should fail. That should happen the way it stands now, so I don't think there's anything more to do there. :+1: